public/js/tests/tests.js

```diff
@@ -1,90 +1,79 @@
-test("should sort channels by name", function(assert) {
-  var channels = [
-    {friendlyName: "BBB"},
-    {friendlyName: "BBA"}
-  ];
+document.body.innerHTML = window.__html__['views/index.jade'];
 
-  var result = twiliochat.sortChannelsByName(channels)
+describe('TwilioChat', function() {
+  it('should sort channels by name', function() {
+    var channels = [
+      {friendlyName: 'BBB'},
+      {friendlyName: 'BBA'}
+    ];
 
-  assert.deepEqual(result, [{friendlyName: "BBA"}, {friendlyName: "BBB"}]);
-});
-
-test("should be able to sort an empty list of channels", function(assert) {
-  var channels = [];
+    var result = twiliochat.sortChannelsByName(channels)
 
-  var result = twiliochat.sortChannelsByName(channels)
+    assert.deepEqual(result, [{friendlyName: 'BBA'}, {friendlyName: 'BBB'}]);
+  });
 
-  assert.deepEqual(result, []);
-});
+  it('should be able to sort an empty list of channels', function() {
+    var channels = [];
 
-test("should sort channels when they have same name", function(assert) {
-  var channels = [
-    {friendlyName: "BBB"},
-    {friendlyName: "BBA"},
-    {friendlyName: "BBA"}
-  ];
+    var result = twiliochat.sortChannelsByName(channels)
 
-  var result = twiliochat.sortChannelsByName(channels);
+    assert.deepEqual(result, []);
+  });
 
-  assert.deepEqual(result, [{friendlyName: "BBA"}, {friendlyName: "BBA"}, {friendlyName: "BBB"}]);
-});
+  it('should sort channels when they have same name', function() {
```

Comment:
This case is already covered by _sorts channels by name_. What about replacing the content of the first case with this content?